File: Homework/homework_04/task2.js
"use strict";

function findNums(start, end) {

    let res = [];
    let arrOfDig;

    for (let i = start; i <= end; i++) {
        // create arr of number digits
        arrOfDig = String(i).split("");

        if (arrOfDig.every(checkEven)) {
            res.push(i);
        }
    }
    if (res.length === 0) { return "Such numbers does not exist"; }
    else { return res; }

    // check if all element is even
    function checkEven(number) {
        return +number % 2 === 0;
    }
}
//test
let arg1 = findNums(19, 42);
let arg2 = findNums(99, 199)
console.log(arg1);
console.log(arg2);

File: Homework/homework_03/archive/task2.py
def get_frequency(li):
    l = len(li)
    res = []
    for x  in list(set(li)):
        k = 0
        for y  in li:
            if x == y: k += 1
        res.append({x: k/l})
    return res

print(get_frequency([1,1,2,2,3]))
print(get_frequency([4,4]))
print(get_frequency([1,2,3]))